src/dispatcher/mod.rs

```diff
@@ -254,14 +277,20 @@ impl Dispatcher {
 
     pub fn down(&mut self, name: String, force: Option<bool>) -> Result<(), ServicingError> {
```

Comment:
BTW. Force here is nothing to do with the flag. It is used to skip the validation flow.

Reply:
oh i see. i didnt realize that, but this logic doesnt solve the issue of skipping the prompt as you have to pass a "-y" to skypilot command.

Reply:
if i run `down` i want to shutdown the service if its up or building. no need to specify `force`